spaces/domains.json

```
@@ -576,4 +576,5 @@
   "vote.wanakafarm.com": "wanakafarm-dao.eth",
   "vote.defikingdoms.com": "dfkvote.eth",
   "dao.astraguild.io": "astraguildventures.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.astraguild.io": "astraguildventures.eth",
```